src/contractTest/java/uk/gov/hmcts/reform/professionalapi/provider/OrganisationalInternalControllerProviderTest.java

```diff
@@ -98,7 +110,7 @@ public void setActiveOrganisationsForLoggedInUser() throws IOException {
             "companyN", false, "www.org.com");
         addSuperUser(organisation);
 
-        when(organisationRepository.findByStatus(OrganisationStatus.ACTIVE)).thenReturn(Arrays.asList(organisation));
+        when(organisationRepository.findByStatus(OrganisationStatus.ACTIVE)).thenReturn(asList(organisation));
 
         ProfessionalUsersEntityResponse professionalUsersEntityResponse = new ProfessionalUsersEntityResponse();
         List<ProfessionalUsersResponse> userProfiles = new ArrayList<>();
```

Comment:
you could use `var` here

Reply:
using `var` here causes issues when `.addAll()` is called later on - as this is not part of my code changes apart from formatting/static import usage - I will leave it as it is